code/modules/jobs/job_exp.dm

```diff
@@ -0,0 +1,203 @@
+// Admin Verbs
+
+/client/proc/cmd_admin_check_player_exp()	//Allows admins to determine who the newer players are.
+	set category = "Admin"
+	set name = "Check Player Playtime"
+	if(!check_rights(R_ADMIN))
+		return
+	var/msg = "<html><head><title>Playtime Report</title></head><body>Playtime:<BR><UL>"
+	for(var/client/C in GLOB.clients)
+		msg += "<LI> - [key_name_admin(C)]: <A href='?_src_=holder;getplaytimewindow=\ref[C.mob]'>" + C.get_exp_living() + "</a></LI>"
+	msg += "</UL></BODY></HTML>"
+	src << browse(msg, "window=Player_playtime_check")
+
+
+/datum/admins/proc/cmd_show_exp_panel(var/client/C)
+	if(!C)
+		to_chat(usr, "ERROR: Client not found.")
+		return
+	if(!check_rights(R_ADMIN))
+		return
+	var/body = "<html><head><title>Playtime for [C.key]</title></head><BODY><BR>Playtime:"
+	body += C.get_exp_report()
+	body += "</BODY></HTML>"
+	usr << browse(body, "window=playerplaytime[C.ckey];size=550x615")
+
+
+// Procs
+
+
+/datum/job/proc/available_in_playtime(client/C)
+	if(!C)
+		return 0
+	if(!exp_requirements || !exp_type)
+		return 0
+	if(!job_is_xp_locked(src.title))
+		return 0
+	if(config.use_exp_restrictions_admin_bypass && check_rights(R_ADMIN, 0, C.mob))
+		return 0
+	var/isexempt = text2num(C.prefs.exp[EXP_TYPE_EXEMPT])
+	if(isexempt)
+		return 0
+	var/my_exp = text2num(C.prefs.exp[get_exp_req_type()])
+	var/job_requirement = text2num(get_exp_req_amount())
+	if(my_exp >= job_requirement)
+		return 0
+	else
+		return (job_requirement - my_exp)
+
+/datum/job/proc/get_exp_req_amount()
+	if(title in GLOB.command_positions)
+		if(config.use_exp_restrictions_heads_hours)
+			return config.use_exp_restrictions_heads_hours * 60
+	return exp_requirements
+
+/datum/job/proc/get_exp_req_type()
+	if(title in GLOB.command_positions)
+		if(config.use_exp_restrictions_heads_department && exp_type_department)
+			return exp_type_department
+	return exp_type
+
+/proc/job_is_xp_locked(jobtitle)
+	if(!config.use_exp_restrictions_heads && jobtitle in GLOB.command_positions)
+		return 0
+	if(!config.use_exp_restrictions_other && !(jobtitle in GLOB.command_positions))
```

Comment:
This is a somewhat old version of the playtime tracking code. I believe I eliminated this from the original code awhile back.